tests/bug01258.phpt

```diff
@@ -0,0 +1,62 @@
+--TEST--
+Test for bug #1258: ensure case statements are covered
+--FILE--
+<?php
+$foo = ['bar', 'baz', 'qux', 'quux'];
+xdebug_start_code_coverage(XDEBUG_CC_UNUSED);
+foreach ($foo as $k => $v) {
+    switch ($v) {
+        case 'bar':
+            echo "bar\n";
+            break;
+        case 'baz':
+            echo "baz\n";
+            break;
+        case 'qux':
+            echo "qux\n";
+            break;
+        default:
+            echo "default\n";
+            break;
+    }
+}
+$cc = xdebug_get_code_coverage();
+xdebug_stop_code_coverage();
+var_dump($cc);
+?>
+--EXPECTF--
+bar
+baz
+qux
+default
+array(1) {
+  ["/home/james/workspace/xdebug/tests/bug01258.php"]=>
```

Comment:
I don't have that directory ;-) I'll fix it before merging until you beat me to it to change it to "%sbug01258.php"